third_party/blink/renderer/core/farbling/brave_session_cache.cc

```diff
@@ -223,11 +223,15 @@ void BraveSessionCache::Init() {
 
 absl::optional<blink::BraveAudioFarblingHelper>
 BraveSessionCache::GetAudioFarblingHelper() {
-  return base::OptionalFromPtr(UpdateAndGetAudioFarblingHelper());
+  if (!farbling_enabled_ || (farbling_level_ == BraveFarblingLevel::OFF))
+    return absl::nullopt;
+  DCHECK(audio_farbling_helper_);
+  return audio_farbling_helper_;
```

Comment:
```
absl::optional<blink::BraveAudioFarblingHelper>
BraveSessionCache::GetAudioFarblingHelper() const {
  return audio_farbling_helper_;
}
```

Reply:
No that's not the same because we were returning null if farbling level is off, but perhaps I can make that adjustment in the constructor.

Reply:
Oh I see your other comment.

Reply:
fixed in https://github.com/brave/brave-core/pull/16264/commits/602c9eb34dde0e7560d8455aa3456df91deaadf9